README.md

```diff
@@ -1,5 +1,45 @@
+
+## Taking it Further
+
+Our documentation also covers advanced topics such as, [managing permissions](http://apostrophecms.org/docs/tutorials/intermediate/permissions.html), [working with Apostrophe data](http://apostrophecms.org/docs/tutorials/intermediate/model-layer.html), [building forms](http://apostrophecms.org/docs/tutorials/intermediate/forms.html), and [hosting an Apostrophe site in production](http://apostrophecms.org/docs/tutorials/intermediate/deployment.html).
 
-# Apostrophe
+Moreover, there's tons of great info in the ["How To"](http://apostrophecms.org/docs/tutorials/howtos/index.html) section of docs, and it's always growing!
 
-Apostrophe is a CMS framework for nodejs that supports in-context editing, schema-driven content types, flexible widgets and a great deal more. This module contains everything necessary to build a website with Apostrophe. Please see the [Apostrophe documentation](http://apostrophenow.org) for complete information about building sites with Apostrophe.
+## Extending the Core with Modules
+
+Apostrophe is Node.js all the way through, which means it can be [extended](http://apostrophecms.org/extend) in all sorts of interesting ways. Here's a list of some of our favorite modules:
+
+* [apostrophe-blog](https://github.com/punkave/apostrophe-blog) - Everything you need to put a blog on your site.
+* [apostrophe-events](https://github.com/punkave/apostrophe-events) - Manage upcoming events.
+* [apostrophe-workflow](https://github.com/punkave/apostrophe-workflow) - Add powerful localization capabilities to your project.
+* [apostrophe-passport](https://github.com/punkave/apostrophe-passport) - Authenticate via Twitter, Facebook, Github and more.
+* [apostrophe-places](https://github.com/punkave/apostrophe-places) - Manage geotagged content and display it on a map.
+* [apostrophe-redirects](https://github.com/punkave/apostrophe-redirects) - An easy redirects module.
+
+## Additional Resources
+We've got some great open-source projects that can help you out deploying, managing and maintaining Apsotrophe sites.
+* [Stagecoach](https://github.com/punkave/stagecoach) - A tool for deploying Node.js web applications to your own servers.
+* [Mechanic](https://github.com/punkave/mechanic) - A tool to manage nginx-powered proxies for Node.js applications.
+
+## Contributing
+If you have an idea for a new or improved feature please [open an issue](https://github.com/punkave/apostrophe/issues). If you come across and edge-case bug, please [add an issue](https://github.com/punkave/apostrophe/issues) for that, as well.
+
+If you'd like to implement that idea or fix that bug, please fork the repository and make the update. Pull requests are warmly welcome.
+
+## License
+Apostrophe is released under the [MIT License](https://github.com/punkave/apostrophe/blob/master/LICENSE.md).
+
+[![Build Status](https://travis-ci.org/punkave/apostrophe.svg?branch=master)](https://travis-ci.org/punkave/apostrophe)
```

Comment:
I would keep build status at the top - also, aren't we not really using travis for this anymore?

Reply:
Heh. That's why I moved it to the bottom. The link showed it was about a month out of date. Maybe I should remove altogether?

Reply:
If we're not using it anymore, it should be removed. We should investigate getting travis working right - I think it adds a degree of credibility.

Reply:
@boutell what's the status of the Travis CI link currently on the README? 

https://travis-ci.org/punkave/apostrophe

Last ran 26 days ago. Should we remove the link or does something need to be done to make sure it runs more regularly?

Reply:
I think we stopped it because the false positives were a bit out of control. There were results I couldn't reproduce. Maybe if we play with the settings more on Travis.

Reply:
Removed. For now.